src/components/Global/SignStatusButton/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Button } from '../index';
+
+const SignStatusButton = ({ status }) => {
+  return (
+    status !== 'authenticated' && (
```

Comment:
This feels so specific for a component, the `'authenticated'` part (the status value in general) depends on logic outside of this file so perhaps we could just conditionally render this button where it's being used?

Reply:
Or maybe just add an extra prop to the `LogStatusButton` component and add this button there?